src/pages/Hero.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box, Button, Heading, Icon, Image } from "@chakra-ui/react";
+import bg from "../res/images/hero.jpg";
+import { MdArrowDownward } from "react-icons/md";
+
+interface HeroProps {
+  scrollToNext: () => void;
+}
+
+const Hero = ({ scrollToNext }: HeroProps) => {
```

Comment:
Why is it named Hero? Does not feel very descriptive

Reply:
A hero image is a website design term used to describe an oversized banner image at the top of a website.